README.md

```diff
@@ -41,7 +41,7 @@ group :test, :development do
 end
 ```
 
-For Ruby < `2.x` and older `FactoryGirl` syntax etc
+For Ruby < `2.x` and older `FactoryBot` syntax etc
```

Comment:
hey sorry, I'm not sure why I didn't see this notification about this PR. I think this change has been requested a few times, and it obviously needs to happen but just in a non-confusing way.

Anyway, I think what probably makes sense here is for your changes to be a major version bump, and for us to add a bit to the README explaining all 3 of the following things:

1. Old old `Factory.define` handled by the current version
2. Old `FactoryGirl` syntax also handled by the current version
3. New `FactoryBot` syntax, handled by the new version.

Then this PR can also bump the major version and I'll cut a release.